frontend/appflowy_flutter/lib/user/presentation/screens/workspace_start_screen/desktop_workspace_start_screen.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:appflowy/generated/locale_keys.g.dart';
```

Comment:
afaik, the desktop workspace page should be deprecated. cc @appflowy

Reply:
From the previous code, I thought the workspace start screen was to let users choose which workspace they are going to or create a new workspace. What is the correct workflow after user login? @appflowy

Reply:
I did some updates [here](https://github.com/AppFlowy-IO/AppFlowy/pull/3217#discussion_r1304940076), and I left `WorkspaceStartScreen` anyway, we may need it to ask the user which workspace they want to open.